app-backend/batch/src/test/resources/export/astJob.json

```diff
@@ -0,0 +1,58 @@
+{
+  "id": "dda6080f-f7ad-455d-b409-764dd8c57039",
+  "input": {
+    "projectId": "d4d99263-dc0e-4555-ad06-e4b22069919c",
+    "resolution": 10,
+    "style": {
+      "ast": {
+        "apply": "+",
+        "args": [
+          {
+            "id": "fea6c736-458a-4b59-b0d2-ac4bf7ba0a13",
+            "metadata": null
+          },
+          {
+            "id": "a928ba7d-1c7b-4988-a546-873bfdb089dd",
+            "metadata": null
+          }
+        ],
+        "id": "370775ee-c8a1-4671-a00a-3a2aa79dfb40",
+        "metadata": null
+      },
+      "params": {
+        "sources": {
+          "fea6c736-458a-4b59-b0d2-ac4bf7ba0a13": {
+            "id": "535f33b5-7a81-4403-b58a-89f6d71faf4a",
+            "band": 5,
+            "type": "scene"
```

Comment:
It doesn't, yet. That could be a good follow-up task, as well as implementing any extra `Operation`s.